tls/s2n_resume.c

```diff
@@ -868,19 +869,22 @@ int s2n_decrypt_session_ticket(struct s2n_connection *conn, struct s2n_stuffer *
     POSIX_GUARD(s2n_stuffer_skip_write(&state_stuffer, state_blob_size));
     POSIX_GUARD_RESULT(s2n_deserialize_resumption_state(conn, &from->blob, &state_stuffer));
 
-    uint64_t now = 0;
-    POSIX_GUARD_RESULT(s2n_config_wall_clock(conn->config, &now));
+    if (s2n_connection_get_protocol_version(conn) == S2N_TLS13) {
```

Comment:
This is a nit, but we technically want to write code that doesn't break if new version of TLS comes out. So this should be:
```suggestion
    if (s2n_connection_get_protocol_version(conn) >= S2N_TLS13) {
```